test/support/setup.js

```diff
@@ -1,8 +1,22 @@
 var Helpers = require('./helpers').Helpers;
 var testHelper = new Helpers();
 
-// Fake API key for tests
-process.env['TWINGLY_SEARCH_KEY'] = testHelper.randomValueHex(16);
+// Do not record new fixtures if one already exists
+process.env['VCR_MODE'] = 'cache';
```

Comment:
Possibly just set it unless set already? As I gather it will not be possible to run in `record` or `playback` mode as we force it to `cache` this way.

Reply:
> As I gather it will not be possible to run in record or playback mode as we force it to cache this way.

You're right, I'll add an "if".